userguide/system.rst

```diff
@@ -2798,8 +2798,8 @@ in the chassis.
 Activating the license adds the :guilabel:`Failover`
 option to :guilabel:`System`. Some fields are modified in
 :guilabel:`Network` so that the peer IP address, peer hostname, and
-virtual IP can be configured. An extra section is added to
-:guilabel:`IPMI` to allow configuring :ref:`IPMI` for each units.
+virtual IP can be configured. An extra drop-down is added to
+:guilabel:`IPMI` to allow configuring :ref:`IPMI` for each |ctrlr-term|.
 Modified fields use *1* or *2* identify the |ctrlrs-term|. These numbers
```

Comment:
Not really clear what this sentence is saying--the "1" or "2" only appears on a field that has been modified?
```suggestion
Modified fields use *1* or *2* to identify the |ctrlrs-term|. These numbers
```